lib/engines/hckinstall/hckinstall.rb

```diff
@@ -235,12 +234,11 @@ def run_first(studio:, client:)
         if studio
           prepare_studio_drives
 
+          iso_list = [@setup_studio_iso, @studio_iso_info['path']]
+          iso_list.push(@kit_path) if @kit_is_iso
```

Comment:
For consistency with the following line:
```suggestion
          iso_list << @kit_path if @kit_is_iso
```